AmazonChimeSDKDemo/AmazonChimeSDKDemoObjC/Base.lproj/Main.storyboard

```diff
@@ -16,136 +17,242 @@
-                            </textField>
-                            <textField opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="left" contentVerticalAlignment="center" borderStyle="roundedRect" placeholder="Your Name" textAlignment="natural" minimumFontSize="17" translatesAutoresizingMaskIntoConstraints="NO" id="GYb-qN-kDH">
-                                <rect key="frame" x="117" y="450" width="180" height="34"/>
-                                <constraints>
-                                    <constraint firstAttribute="width" constant="180" id="G50-Ai-0VW"/>
-                                </constraints>
-                                <fontDescription key="fontDescription" type="system" pointSize="14"/>
-                                <textInputTraits key="textInputTraits"/>
-                            </textField>
-                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Join a Meeting" textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="uia-Yo-wKO">
-                                <rect key="frame" x="82.5" y="320" width="249.5" height="48"/>
-                                <fontDescription key="fontDescription" type="system" pointSize="40"/>
-                                <nil key="textColor"/>
-                                <nil key="highlightedColor"/>
-                            </label>
-                            <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="WLF-PZ-6RX">
-                                <rect key="frame" x="195" y="500" width="24" height="24"/>
-                                <constraints>
-                                    <constraint firstAttribute="width" constant="24" id="H3a-gB-JqH"/>
-                                    <constraint firstAttribute="height" constant="24" id="oQD-n5-vM8"/>
-                                </constraints>
-                                <state key="normal" image="join"/>
-                                <connections>
-                                    <action selector="joinMeeting:" destination="BYZ-38-t0r" eventType="touchUpInside" id="h3b-1e-S7J"/>
-                                </connections>
-                            </button>
-                            <button hidden="YES" opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="yRo-PN-JIQ">
-                                <rect key="frame" x="195" y="500" width="24" height="24"/>
+                            </view>
```

Comment:
Since the UI changed, were you able to provide the flow of screen shots like what happens when end button is pressed, return back to join screen?